server.js

```diff
@@ -3452,6 +3452,53 @@ cache(function(data, match, sendBadge, request) {
     }
   });
 }));
+// Magnum CI integration
+camp.route(/^\/magnumci\/([^\/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you please add a postfix refining what the badge does, to avoid collision with a potential future other magnumci badge? Maybe something like `/magnumci/t/` for "tests". Or maybe `/magnum/ci/`? Your call.